.github/workflows/conformance.yml

```diff
@@ -29,7 +29,6 @@ jobs:
     - uses: sigstore/sigstore-conformance@6bd1c54e236c9517da56f7344ad16cc00439fe19 # v0.0.13
       with:
         entrypoint: ${{ github.workspace }}/packages/conformance/bin/run
-        xfail: "test_verify_with_trust_root"
```

Comment:
We no longer need to skip this test now that we support all the flags on the `verify` command.